### DISCUSSION TITLE
Scrollable component

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#8224] [Help]
I have created Custom component for displaying data from the database for getting all data from table I need to scroll full screen but I don't want scroll whole screen 
is there any way where I need to scroll In any specific component

**Comment by melohagan** [comment]
Sounds to me that pagination might be useful to you? It is controlled by the data provider, so if you have a custom component that supports data providers then the pagination is already done for you:

 - https://docs.budibase.com/docs/data-provider#pagination-and-sorting
 - https://docs.budibase.com/docs/custom-component#example-data-provider
